### PR TITLE
Create metamorphpro

### PR DESCRIPTION
**Pull request by danlewcom**
**Add space for ...**  
  
Etherscan: https://etherscan.io/token/0xfef3884b603c33ef8ed4183346e093a173c94da6
CoinGecko: https://www.coingecko.com/en/coins/metamorph
Website: https://metamorph.pro
Twitter: https://twitter.com/metamorphpro
logo
space

### CODE REVIEW

File: spaces/metamorphpro [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+"address": "0xfef3884b603c33ef8ed4183346e093a173c94da6", // The address of the base token
+"token": "0xfef3884b603c33ef8ed4183346e093a173c94da6", // The same address of the base token
+"core": [ // List of official addresses that can post in "Core" tab of your space
+"0x7A9dbEAd85731D7ec80D28e63aE6bACCA99F948E"
+],
+"min": 123, // Minimum balance to have from the base token to have your proposal visible in the space (unless the address is a core address)
+"invalid": [
+"0x7A9dbEAd85731D7ec80D28e63aE6bACCA99F948E"
+] // List of proposal ids, use this to remove a proposal from your space
+}
```
Comment (e-asia) [inline spaces/metamorphpro:17]: You should reformat code and delete comments.
Both images `logo.png` and `space.png` should be in your `spaces/metamorphpro` directory and should not exceed 50kb.
Thank you.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/metamorphpro (hunk 1 above)
Comment (e-asia) [reply spaces/metamorphpro:17]: Your images still missing in your PR.

Comment (danlewcom) [reply spaces/metamorphpro:17]: space

**Review by danlewcom — COMMENTED** [review]
Please approve, I have added both space.png and logo.png

File: spaces/metamorphpro (hunk 1 above)
Comment (e-asia) [reply spaces/metamorphpro:17]: You shouldn’t upload your files anywhere but add it with git command line and commit it in your Pull Request.
Something like:
`
git add spaces/metamorphpro/logo.png spaces/metamorphpro/space.png
git commit -m "add spaces/metamorphpro images"
git push
`

**Comment by e-asia** [comment]
We have updated all the spaces. You have to do a git pull before commit your changes again or create a new pull request.

**Pull request closed without merging**